src/dgisim/encoding/encoding_plan.py

```diff
@@ -261,6 +262,13 @@ def perspective_version(self, pid: Pid) -> Self:
             self._flip_cache = new_self
         return self._flip_cache
 
+class EncodingPlanFast(EncodingPlan):
```

Comment:
I prefer using the name `LazyEncodingPlan`, as `EncodingPlanFast` suggests it is just a faster version (without hinting it has different functionalities).

Reply:
What you said is absolutely correct. The above code has all been modified.